operations/commit_queue.go

```diff
@@ -176,7 +176,12 @@ func mergeCommand() cli.Command {
 			client := conf.setupRestCommunicator(ctx)
 			defer client.Close()
 
-			return params.mergeBranch(ctx, conf, client, ac)
+			settings, err := client.GetSettings(ctx)
+			if err != nil {
+				return errors.Wrap(err, "problem retrieving admin settings")
+			}
+
+			return params.mergeBranch(ctx, conf, client, ac, settings.Ui.UIv2Url)
```

Comment:
I'm not sure there's a reason to get UIv2Url before entering mergeBranch since we're passing in client and ctx; can we just retrieve it at the time it's necessary?